exporter/exporter_test.go

```diff
@@ -27,7 +27,7 @@ func TestExport(t *testing.T) {
 
 	startDate := int64(1451606400)
 	endDate := int64(1483228799)
-	fmt.Print(GenerateCat1(patientData, measureData, valueSetData, startDate, endDate))
+	fmt.Print(GenerateCat1(patientData, measureData, valueSetData, startDate, endDate, "r3"))
```

Comment:
That test doesn't actually assert anything, so we should be okay for now. It just spits out a copy of the XML.